tests/osgDB/Input.js

```diff
@@ -203,13 +203,15 @@ module.exports = function () {
                 'mode': 'TRIANGLES'
             }
         } );
-        input.readPrimitiveSet().then( function ( /*value */) {
+        input.readPrimitiveSet().then( function ( /* value */) {
             return input.setJSON( {
                 'DrawArrays': {
                     'UniqueID': 10
                 }
             } ).readPrimitiveSet();
         } ).then( function ( o2 ) {
+            console.log( '>>>' );
```

Comment:
You could probably squash all commits into one. Also keep commits message clear and not too long like this https://github.com/cedricpinson/osgjs/pull/796/commits/d50f1750b5b1ed4e3a002dd9a6765ee43ad7dee7
If you need to add details or explanation do it after the first line of the commit message

but it's good, just worry about performance